repos/system_upgrade/common/actors/checkselinux/libraries/checkselinux.py

```diff
@@ -0,0 +1,69 @@
+from leapp import reporting
+from leapp.libraries.common.config.version import get_target_major_version
+from leapp.libraries.stdlib import api
+from leapp.models import KernelCmdlineArg, SELinuxFacts, SelinuxPermissiveDecision, SelinuxRelabelDecision
+
+DOC_URL = 'https://red.ht/rhel9-disabling-selinux'
+
+
+def process():
+    facts = next(api.consume(SELinuxFacts), None)
+    if not facts:
+        return
+
+    enabled = facts.enabled
+    conf_status = facts.static_mode
+
+    if conf_status == 'disabled':
+        if get_target_major_version() == '9':
+            api.produce(KernelCmdlineArg(key='selinux', value='0'))
+            reporting.create_report([
+                reporting.Title('LEAPP detected SELinux disabled in "/etc/selinux/config"'),
+                reporting.Summary(
+                    'On RHEL 9, disabling SELinux in "/etc/selinux/config" is no longer possible. '
+                    'This way, the system starts with SELinux enabled but with no policy loaded. LEAPP '
+                    'will automatically disable SELinux using "SELINUX=0" kernel command line parameter. '
+                    'However, Red Hat strongly recommends to have SELinux enabled'
+                ),
+                reporting.Severity(reporting.Severity.INFO),
+                reporting.Tags([reporting.Tags.SELINUX]),
+                reporting.RelatedResource('file', '/etc/selinux/config'),
+                reporting.ExternalLink(url=DOC_URL, title='Disabling SELinux'),
+            ])
+
+        if enabled:
+            reporting.create_report([
+                reporting.Title('SElinux disabled in configuration file but currently enabled'),
+                reporting.Summary('This message is to inform user about non-standard SElinux configuration.'),
```

Comment:
this is bad msg. the summary basically is not telling user why the situation happened and what they are expected to do.

Reply:
This is not change of this PR. It was just refactored into library. Wondering whether we want to change a report message which is already out but, mmm, why not...